cmd/profile.go

```diff
@@ -10,9 +10,15 @@ import (
 	"github.com/spf13/cobra"
 )
 
+var (
+	runningInLinux         = runtime.GOOS == "linux"
+	runningInLinuxTestMode = runningInLinux && config.TestMode == "true"
+)
+
 var profileCmd = &cobra.Command{
-	Use:   "profile",
-	Short: "Manage profiles (experimental)",
+	Use:    "profile",
+	Short:  "Manage profiles (experimental)",
+	Hidden: runningInLinux && !runningInLinuxTestMode,
```

Comment:
nit: considering `(A and not(A and B))` to resolve `not(A and B)` a must be true so this would be `(A and not(true and B))` which is the same as `(A and not(B))`